e2e/specs/create.workspace.spec.js

```diff
@@ -17,4 +17,6 @@ describe('pivotal tracker page', () => {
         workspace = dashboard.createWorkspace(workspaceName);
         expect(workspaceName).to.equal(workspace.getWorkspaceName());
     });
+
+    //TODO after delete workspace from api
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS1135)